modules/luci-mod-network/htdocs/luci-static/resources/view/network/switch.js

```diff
@@ -299,7 +297,11 @@ return view.extend({
 				return true;
 			};
 
+			o.port_opts = [];
+			var port_opts = o.port_opts;
+
 			o.write = function(section_id, value) {
+				var port_opts = this.port_opts;
 				var topology = this.section.topology,
 				    values = [];
 
```

Comment:
... and change all subsequent uses of `port_opts` within the `o.write()` function scope to `this.port_opts`

Reply:
done

Reply:
Perfect, thank you!